.github/CODEOWNERS

```diff
@@ -77,11 +77,12 @@ subprojects/docs/src/docs/userguide/reference/directory_layout.adoc
 subprojects/docs/src/docs/userguide/troubleshooting/version_catalog_problems.adoc @gradle/bt-jvm
 
 # Dev-prod team
-.teamcity/                                  @gradle/bt-developer-productivity
-.github/                                    @gradle/bt-developer-productivity
-gradle/shared-with-buildSrc/                @gradle/bt-developer-productivity
-build-logic/                                @gradle/bt-developer-productivity
-build-logic-commons/                        @gradle/bt-developer-productivity
-build-logic-settings/                       @gradle/bt-developer-productivity
-subprojects/*/build.gradle*                 @gradle/bt-developer-productivity
-*.gradle.kts                                @gradle/bt-developer-productivity
+.teamcity/                   @gradle/bt-developer-productivity
+.github/                     @gradle/bt-developer-productivity
+gradle/shared-with-buildSrc/ @gradle/bt-developer-productivity
+build-logic/                 @gradle/bt-developer-productivity
+build-logic-commons/         @gradle/bt-developer-productivity
+build-logic-settings/        @gradle/bt-developer-productivity
+subprojects/*/build.gradle*  @gradle/bt-developer-productivity
+build.gradle*                @gradle/bt-developer-productivity
+settings.gradle*              @gradle/bt-developer-productivity
```

Comment:
We can try this, though I don't expect any changes in behavior.

Reply:
I exect to not have reviews assigned from docs samples, like:
 - [#24021](https://github.com/gradle/gradle/pull/24021/files?file-filters%5B%5D=.gradle)
 - [#23685](https://github.com/gradle/gradle/pull/23685/files?file-filters%5B%5D=.gradle)